client/scripts/views/components/widgets/user.ts

```diff
@@ -55,18 +56,18 @@ const User: m.Component<{
         account = app.chain.accounts.get(address);
       }
       profile = app.profiles.getProfile(chainId, address);
-      role = app.user.isAdminOrMod({ account: vnode.attrs.user });
+      role = adminsAndMods.find((r) => r.address === address);
     } else {
       account = vnode.attrs.user;
       profile = app.profiles.getProfile(account.chain.id, account.address);
-      role = app.user.isAdminOrMod({ account });
+      role = adminsAndMods.find((r) => r.address === account.address);
```

Comment:
Can we use the account comparison helper here? It's possible someone might have the same address on two chains.

If not, we should compare both address and chain.

Reply:
The comparison helpers we have are attached to the RoleController (on the UserController) which means the roles it has access to are only those of the logged-in user. 

I changed the check above to include a verification of chain by address:
`role = adminsAndMods.find((r) => r.address === address && r.address_chain === chainId);`
`role = adminsAndMods.find((r) => r.address === account.address && r.address_chain == account.chain.id);`